src/test/skript/tests/syntaxes/effects/EffToggleCanPickUpItems.sk

```diff
@@ -0,0 +1,8 @@
+test "entity can pick up items":
+	spawn a zombie at spawn of "world"
+	set {_zombie} to last spawned zombie
+	let {_zombie} pick up items
+	assert {_zombie} can pick up items with "failed to allow zombie to pick up items"
+	forbid {_zombie} from picking items up
+	assert {_zombie} can't pick up items with "failed to disallow zombie to pick up items"
+	delete {_zombie}
```

Comment:
This doesn't actually delete the zombie within the variable but rather the variable itself, you could use `delete random entity of {_zombie}`

Reply:
good catch thanks